pkg/test/resourcefixture/testdata/basic/bigtable/v1beta1/bigtableappprofile/_generated_object_bigtableappprofile.golden.yaml

```diff
@@ -32,5 +30,5 @@ status:
     reason: UpToDate
     status: "True"
     type: Ready
-  name: projects/${projectId}/instances/profiledep${uniqueId}/appProfiles/bigtableappprofile-${uniqueId}
```

Comment:
We cannot remove "name" for backward compatibility reasons. This field needs to be filled after CREATE and UPDATE.